### DISCUSSION TITLE
Cannot create a nx workspace for angular.  Getting cypress error cannot be inzipped

### DISCUSSION THREAD

**Discussion by scriptlord** [cypress-io/cypress#25268] [Questions and Help]
yarn run v1.22.19
$ /private/var/folders/kr/qwjdryyn7q1cr88kwg4tyvnw0000gn/T/tmp-6674-7ggH900YPSsL/node_modules/.bin/nx new apppp --no-allPrompts --no-a --no-skipGit --no-g '--commit.message=Initial commit' --/bin/sh=/usr/local/bin/create-nx-workspace --name=apppp --preset=angular-monorepo --appName=sappppp --style=scss --no-nxCloud --packageManager=yarn --defaultBase=main --ci= --nxWorkspaceRoot=/Users/mac
info No lockfile found.
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
success Saved lockfile.
$ node ./decorate-angular-cli.js

 >  NX   Decoration of the Angular CLI did not complete successfully

$ /Users/mac/apppp/node_modules/.bin/nx g @nrwl/workspace:preset --name=sappppp --style=scss --linter=eslint --npmScope=apppp --preset=angular-monorepo --packageManager=yarn --interactive=true

>  NX  Generating @nrwl/workspace:preset

UPDATE nx.json
UPDATE package.json
CREATE jest.preset.js
CREATE jest.config.ts
UPDATE .vscode/extensions.json
UPDATE .gitignore
CREATE apps/sappppp/tsconfig.app.json
CREATE apps/sappppp/tsconfig.spec.json
CREATE apps/sappppp/src/favicon.ico
CREATE apps/sappppp/src/index.html
CREATE apps/sappppp/src/main.ts
CREATE apps/sappppp/src/styles.scss
CREATE apps/sappppp/src/assets/.gitkeep
CREATE apps/sappppp/src/app/app.module.ts
CREATE apps/sappppp/src/app/app.component.scss
CREATE apps/sappppp/src/app/app.component.html
CREATE apps/sappppp/src/app/app.component.spec.ts
CREATE apps/sappppp/src/app/app.component.ts
CREATE apps/sappppp/project.json
CREATE apps/sappppp/tsconfig.editor.json
CREATE apps/sappppp/tsconfig.json
CREATE apps/sappppp/src/app/nx-welcome.component.ts
CREATE .eslintrc.json
CREATE apps/sappppp/.eslintrc.json
CREATE apps/sappppp/jest.config.ts
CREATE apps/sappppp/src/test-setup.ts
CREATE apps/sappppp-e2e/cypress.config.ts
CREATE apps/sappppp-e2e/src/e2e/app.cy.ts
CREATE apps/sappppp-e2e/src/fixtures/example.json
CREATE apps/sappppp-e2e/src/support/app.po.ts
CREATE apps/sappppp-e2e/src/support/commands.ts
CREATE apps/sappppp-e2e/src/support/e2e.ts
CREATE apps/sappppp-e2e/tsconfig.json
CREATE apps/sappppp-e2e/project.json
CREATE apps/sappppp-e2e/.eslintrc.json
[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
[4/4] Building fresh packages...
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

 >  NX   Command failed: yarn

   Pass --verbose to see the stacktrace.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

 >  NX   Workspace creation failed, see above.

   Pass --verbose to see the stacktrace.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

warning @nrwl/angular > magic-string > sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
warning @nrwl/angular > @nguniversal/builders > guess-parser > @wessberg/ts-evaluator@0.0.27: this package has been renamed to ts-evaluator. Please install ts-evaluator instead
warning @nrwl/angular > @nguniversal/builders > guess-parser > @wessberg/ts-evaluator > jsdom > w3c-hr-time@1.0.2: Use your platform's native performance.now() and performance.timeOrigin.
warning @nrwl/angular > @nrwl/webpack > stylus > css > source-map-resolve@0.6.0: See https://github.com/lydell/source-map-resolve#deprecated
warning @nrwl/angular > @nrwl/webpack > css-minimizer-webpack-plugin > cssnano > cssnano-preset-default > postcss-svgo > svgo > stable@0.1.8: Modern JS already guarantees Array#sort() is a stable sort, so this library is deprecated. See the compatibility table on MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#browser_compatibility
warning "@nrwl/angular > @nguniversal/builders@15.0.0" has unmet peer dependency "@angular-devkit/build-angular@^15.0.0-next.0".
warning "@nrwl/angular > ts-node@10.9.1" has unmet peer dependency "@types/node@*".
warning "@nrwl/angular > @nguniversal/builders > @nguniversal/common@15.0.0" has unmet peer dependency "@angular/common@^15.0.0-next.0".
warning "@nrwl/angular > @nguniversal/builders > @nguniversal/common@15.0.0" has unmet peer dependency "@angular/core@^15.0.0-next.0".
warning "@nrwl/angular > @nrwl/webpack > babel-loader@8.3.0" has unmet peer dependency "@babel/core@^7.0.0".
[NX] Angular devkit called `writeWorkspace`, this may have created 'workspace.json' or 'angular.json
[NX] Double check workspace configuration before proceeding
Skipping sappppp since apps/sappppp/project.json already exists.
warning @angular/cli > pacote > @npmcli/run-script > node-gyp > make-fetch-happen > cacache > @npmcli/move-file@2.0.1: This functionality has been moved to @npmcli/fs
warning "@nrwl/angular > @nrwl/webpack > babel-loader@8.3.0" has unmet peer dependency "@babel/core@^7.0.0".
error /Users/mac/apppp/node_modules/cypress: Command failed.
Exit code: 1
Command: node index.js --exec install
Arguments: 
Directory: /Users/mac/apppp/node_modules/cypress
Output:
Installing Cypress (version: 11.2.0)

[STARTED] Task without title.
[TITLE]  Downloading Cypress      0% 0s
[TITLE]  Downloading Cypress      1% 112s
[TITLE]  Downloading Cypress      1% 122s
[TITLE]  Downloading Cypress      1% 132s
[TITLE]  Downloading Cypress      1% 142s
[TITLE]  Downloading Cypress      1% 152s
[TITLE]  Downloading Cypress      1% 196s
[TITLE]  Downloading Cypress      1% 207s
[TITLE]  Downloading Cypress      1% 231s
[TITLE]  Downloading Cypress      1% 241s
[TITLE]  Downloading Cypress      1% 297s
[TITLE]  Downloading Cypress      1% 307s
[TITLE]  Downloading Cypress      1% 322s
[TITLE]  Downloading Cypress      1% 333s
[TITLE]  Downloading Cypress      1% 347s
[TITLE]  Downloading Cypress      2% 177s
[TITLE]  Downloading Cypress      2% 188s
[TITLE]  Downloading Cypress      2% 193s
[TITLE]  Downloading Cypress      2% 198s
[TITLE]  Downloading Cypress      2% 203s
[TITLE]  Downloading Cypress      2% 208s
[TITLE]  Downloading Cypress      2% 213s
[TITLE]  Downloading Cypress      2% 218s
[TITLE]  Downloading Cypress      2% 223s
[TITLE]  Downloading Cypress      3% 150s
[TITLE]  Downloading Cypress      3% 153s
[TITLE]  Downloading Cypress      3% 157s
[TITLE]  Downloading Cypress      3% 160s
[TITLE]  Downloading Cypress      3% 165s
[TITLE]  Downloading Cypress      3% 168s
[TITLE]  Downloading Cypress      3% 171s
[TITLE]  Downloading Cypress      3% 174s
[TITLE]  Downloading Cypress      3% 177s
[TITLE]  Downloading Cypress      3% 181s
[TITLE]  Downloading Cypress      3% 184s
[TITLE]  Downloading Cypress      3% 187s
[TITLE]  Downloading Cypress      3% 190s
[TITLE]  Downloading Cypress      4% 144s
[TITLE]  Downloading Cypress      4% 146s
[TITLE]  Downloading Cypress      4% 149s
[TITLE]  Downloading Cypress      4% 151s
[TITLE]  Downloading Cypress      4% 153s
[TITLE]  Downloading Cypress      4% 156s
[TITLE]  Downloading Cypress      4% 163s
[TITLE]  Downloading Cypress      4% 165s
[TITLE]  Downloading Cypress      4% 167s
[TITLE]  Downloading Cypress      4% 178s
[TITLE]  Downloading Cypress      4% 181s
[TITLE]  Downloading Cypress      4% 194s
[TITLE]  Downloading Cypress      5% 156s
[TITLE]  Downloading Cypress      5% 158s
[TITLE]  Downloading Cypress      5% 160s
[TITLE]  Downloading Cypress      5% 162s
[TITLE]  Downloading Cypress      5% 163s
[TITLE]  Downloading Cypress      5% 165s
[TITLE]  Downloading Cypress      5% 167s
[TITLE]  Downloading Cypress      6% 140s
[TITLE]  Downloading Cypress      6% 141s
[TITLE]  Downloading Cypress      6% 143s
[TITLE]  Downloading Cypress      6% 144s
[TITLE]  Downloading Cypress      6% 147s
[TITLE]  Downloading Cypress      6% 148s
[TITLE]  Downloading Cypress      6% 150s
[TITLE]  Downloading Cypress      6% 151s
[TITLE]  Downloading Cypress      6% 153s
[TITLE]  Downloading Cypress      6% 155s
[TITLE]  Downloading Cypress      6% 156s
[TITLE]  Downloading Cypress      6% 158s
[TITLE]  Downloading Cypress      6% 159s
[TITLE]  Downloading Cypress      7% 137s
[TITLE]  Downloading Cypress      7% 138s
[TITLE]  Downloading Cypress      7% 140s
[TITLE]  Downloading Cypress      7% 144s
[TITLE]  Downloading Cypress      7% 145s
[TITLE]  Downloading Cypress      7% 146s
[TITLE]  Downloading Cypress      7% 148s
[TITLE]  Downloading Cypress      7% 149s
[TITLE]  Downloading Cypress      7% 150s
[TITLE]  Downloading Cypress      7% 152s
[TITLE]  Downloading Cypress      7% 164s
[TITLE]  Downloading Cypress      7% 165s
[TITLE]  Downloading Cypress      7% 167s
[TITLE]  Downloading Cypress      7% 168s
[TITLE]  Downloading Cypress      8% 146s
[TITLE]  Downloading Cypress      8% 148s
[TITLE]  Downloading Cypress      8% 149s
[TITLE]  Downloading Cypress      8% 157s
[TITLE]  Downloading Cypress      8% 158s
[TITLE]  Downloading Cypress      8% 162s
[TITLE]  Downloading Cypress      8% 164s
[TITLE]  Downloading Cypress      8% 170s
[TITLE]  Downloading Cypress      8% 172s
[TITLE]  Downloading Cypress      8% 175s
[TITLE]  Downloading Cypress      8% 176s
[TITLE]  Downloading Cypress      8% 180s
[TITLE]  Downloading Cypress      8% 182s
[TITLE]  Downloading Cypress      8% 190s
[TITLE]  Downloading Cypress      8% 191s
[TITLE]  Downloading Cypress      8% 192s
[TITLE]  Downloading Cypress      8% 199s
[TITLE]  Downloading Cypress      8% 200s
[TITLE]  Downloading Cypress      9% 180s
[TITLE]  Downloading Cypress      9% 181s
[TITLE]  Downloading Cypress      9% 182s
[TITLE]  Downloading Cypress      9% 186s
[TITLE]  Downloading Cypress      9% 187s
[TITLE]  Downloading Cypress      9% 188s
[TITLE]  Downloading Cypress      9% 189s
[TITLE]  Downloading Cypress      9% 190s
[TITLE]  Downloading Cypress      9% 191s
[TITLE]  Downloading Cypress      9% 192s
[TITLE]  Downloading Cypress      9% 201s
[TITLE]  Downloading Cypress      10% 180s
[TITLE]  Downloading Cypress      10% 181s
[TITLE]  Downloading Cypress      10% 182s
[TITLE]  Downloading Cypress      10% 183s
[TITLE]  Downloading Cypress      10% 184s
[TITLE]  Downloading Cypress      10% 185s
[TITLE]  Downloading Cypress      10% 194s
[TITLE]  Downloading Cypress      10% 195s
[TITLE]  Downloading Cypress      10% 197s
[TITLE]  Downloading Cypress      10% 198s
[TITLE]  Downloading Cypress      10% 199s
[TITLE]  Downloading Cypress      10% 200s
[TITLE]  Downloading Cypress      10% 201s
[TITLE]  Downloading Cypress      10% 202s
[TITLE]  Downloading Cypress      10% 203s
[TITLE]  Downloading Cypress      10% 204s
[TITLE]  Downloading Cypress      10% 205s
[TITLE]  Downloading Cypress      10% 206s
[TITLE]  Downloading Cypress      10% 207s
[TITLE]  Downloading Cypress      10% 208s
[TITLE]  Downloading Cypress      10% 209s
[TITLE]  Downloading Cypress      10% 210s
[TITLE]  Downloading Cypress      10% 211s
[TITLE]  Downloading Cypress      10% 212s
[TITLE]  Downloading Cypress      10% 213s
[TITLE]  Downloading Cypress      10% 214s
[TITLE]  Downloading Cypress      11% 193s
[TITLE]  Downloading Cypress      11% 194s
[TITLE]  Downloading Cypress      11% 195s
[TITLE]  Downloading Cypress      11% 196s
[TITLE]  Downloading Cypress      11% 197s
[TITLE]  Downloading Cypress      11% 198s
[TITLE]  Downloading Cypress      11% 199s
[TITLE]  Downloading Cypress      11% 200s
[TITLE]  Downloading Cypress      11% 201s
[TITLE]  Downloading Cypress      11% 204s
[TITLE]  Downloading Cypress      11% 205s
[TITLE]  Downloading Cypress      11% 207s
[TITLE]  Downloading Cypress      11% 208s
[TITLE]  Downloading Cypress      11% 210s
[TITLE]  Downloading Cypress      11% 211s
[TITLE]  Downloading Cypress      11% 212s
[TITLE]  Downloading Cypress      11% 213s
[TITLE]  Downloading Cypress      11% 214s
[TITLE]  Downloading Cypress      11% 215s
[TITLE]  Downloading Cypress      12% 196s
[TITLE]  Downloading Cypress      12% 197s
[TITLE]  Downloading Cypress      12% 198s
[TITLE]  Downloading Cypress      12% 199s
[TITLE]  Downloading Cypress      12% 200s
[TITLE]  Downloading Cypress      12% 201s
[TITLE]  Downloading Cypress      12% 202s
[TITLE]  Downloading Cypress      12% 203s
[TITLE]  Downloading Cypress      12% 209s
[TITLE]  Downloading Cypress      12% 210s
[TITLE]  Downloading Cypress      12% 212s
[TITLE]  Downloading Cypress      12% 213s
[TITLE]  Downloading Cypress      12% 217s
[TITLE]  Downloading Cypress      12% 218s
[TITLE]  Downloading Cypress      12% 219s
[TITLE]  Downloading Cypress      12% 220s
[TITLE]  Downloading Cypress      12% 221s
[TITLE]  Downloading Cypress      13% 202s
[TITLE]  Downloading Cypress      13% 203s
[TITLE]  Downloading Cypress      13% 204s
[TITLE]  Downloading Cypress      13% 210s
[TITLE]  Downloading Cypress      13% 211s
[TITLE]  Downloading Cypress      13% 213s
[TITLE]  Downloading Cypress      13% 214s
[TITLE]  Downloading Cypress      13% 217s
[TITLE]  Downloading Cypress      13% 218s
[TITLE]  Downloading Cypress      13% 219s
[TITLE]  Downloading Cypress      13% 220s
[TITLE]  Downloading Cypress      13% 221s
[TITLE]  Downloading Cypress      13% 222s
[TITLE]  Downloading Cypress      13% 223s
[TITLE]  Downloading Cypress      13% 225s
[TITLE]  Downloading Cypress      14% 207s
[TITLE]  Downloading Cypress      14% 208s
[TITLE]  Downloading Cypress      14% 209s
[TITLE]  Downloading Cypress      14% 210s
[TITLE]  Downloading Cypress      14% 211s
[TITLE]  Downloading Cypress      14% 215s
[TITLE]  Downloading Cypress      14% 216s
[TITLE]  Downloading Cypress      14% 217s
[TITLE]  Downloading Cypress      14% 222s
[TITLE]  Downloading Cypress      14% 223s
[TITLE]  Downloading Cypress      14% 224s
[TITLE]  Downloading Cypress      14% 225s
[TITLE]  Downloading Cypress      14% 228s
[TITLE]  Downloading Cypress      14% 229s
[TITLE]  Downloading Cypress      14% 231s
[TITLE]  Downloading Cypress      14% 232s
[TITLE]  Downloading Cypress      15% 215s
[TITLE]  Downloading Cypress      15% 216s
[TITLE]  Downloading Cypress      15% 217s
[TITLE]  Downloading Cypress      15% 218s
[TITLE]  Downloading Cypress      15% 219s
[TITLE]  Downloading Cypress      15% 220s
[TITLE]  Downloading Cypress      15% 221s
[TITLE]  Downloading Cypress      15% 222s
[TITLE]  Downloading Cypress      15% 223s
[TITLE]  Downloading Cypress      15% 224s
[TITLE]  Downloading Cypress      15% 225s
[TITLE]  Downloading Cypress      15% 226s
[TITLE]  Downloading Cypress      15% 227s
[TITLE]  Downloading Cypress      15% 230s
[TITLE]  Downloading Cypress      15% 231s
[TITLE]  Downloading Cypress      15% 232s
[TITLE]  Downloading Cypress      16% 215s
[TITLE]  Downloading Cypress      16% 216s
[TITLE]  Downloading Cypress      16% 219s
[TITLE]  Downloading Cypress      16% 220s
[TITLE]  Downloading Cypress      16% 221s
[TITLE]  Downloading Cypress      16% 224s
[TITLE]  Downloading Cypress      16% 225s
[TITLE]  Downloading Cypress      16% 227s
[TITLE]  Downloading Cypress      16% 230s
[TITLE]  Downloading Cypress      17% 214s
[TITLE]  Downloading Cypress      17% 216s
[TITLE]  Downloading Cypress      17% 218s
[TITLE]  Downloading Cypress      17% 220s
[TITLE]  Downloading Cypress      17% 221s
[TITLE]  Downloading Cypress      17% 228s
[TITLE]  Downloading Cypress      17% 229s
[TITLE]  Downloading Cypress      17% 230s
[TITLE]  Downloading Cypress      18% 215s
[TITLE]  Downloading Cypress      18% 217s
[TITLE]  Downloading Cypress      18% 218s
[TITLE]  Downloading Cypress      18% 220s
[TITLE]  Downloading Cypress      18% 221s
[TITLE]  Downloading Cypress      18% 223s
[TITLE]  Downloading Cypress      18% 224s
[TITLE]  Downloading Cypress      18% 226s
[TITLE]  Downloading Cypress      18% 227s
[TITLE]  Downloading Cypress      18% 228s
[TITLE]  Downloading Cypress      18% 230s
[TITLE]  Downloading Cypress      18% 231s
[TITLE]  Downloading Cypress      18% 234s
[TITLE]  Downloading Cypress      18% 235s
[TITLE]  Downloading Cypress      18% 237s
[TITLE]  Downloading Cypress      19% 222s
[TITLE]  Downloading Cypress      19% 223s
[TITLE]  Downloading Cypress      19% 224s
[TITLE]  Downloading Cypress      19% 225s
[TITLE]  Downloading Cypress      19% 226s
[TITLE]  Downloading Cypress      19% 227s
[TITLE]  Downloading Cypress      19% 228s
[TITLE]  Downloading Cypress      19% 229s
[TITLE]  Downloading Cypress      19% 230s
[TITLE]  Downloading Cypress      20% 216s
[TITLE]  Downloading Cypress      20% 217s
[TITLE]  Downloading Cypress      20% 218s
[TITLE]  Downloading Cypress      20% 219s
[TITLE]  Downloading Cypress      20% 220s
[TITLE]  Downloading Cypress      20% 221s
[TITLE]  Downloading Cypress      20% 222s
[TITLE]  Downloading Cypress      20% 223s
[TITLE]  Downloading Cypress      21% 210s
[TITLE]  Downloading Cypress      21% 211s
[TITLE]  Downloading Cypress      21% 212s
[TITLE]  Downloading Cypress      21% 214s
[TITLE]  Downloading Cypress      21% 215s
[TITLE]  Downloading Cypress      21% 216s
[TITLE]  Downloading Cypress      21% 217s
[TITLE]  Downloading Cypress      21% 220s
[TITLE]  Downloading Cypress      21% 221s
[TITLE]  Downloading Cypress      21% 222s
[TITLE]  Downloading Cypress      21% 223s
[TITLE]  Downloading Cypress      21% 224s
[TITLE]  Downloading Cypress      22% 212s
[TITLE]  Downloading Cypress      22% 213s
[TITLE]  Downloading Cypress      22% 214s
[TITLE]  Downloading Cypress      22% 215s
[TITLE]  Downloading Cypress      22% 216s
[TITLE]  Downloading Cypress      22% 217s
[TITLE]  Downloading Cypress      22% 219s
[TITLE]  Downloading Cypress      22% 221s
[TITLE]  Downloading Cypress      23% 209s
[TITLE]  Downloading Cypress      23% 210s
[TITLE]  Downloading Cypress      23% 211s
[TITLE]  Downloading Cypress      23% 212s
[TITLE]  Downloading Cypress      23% 213s
[TITLE]  Downloading Cypress      24% 201s
[TITLE]  Downloading Cypress      24% 202s
[TITLE]  Downloading Cypress      24% 203s
[TITLE]  Downloading Cypress      24% 204s
[TITLE]  Downloading Cypress      24% 205s
[TITLE]  Downloading Cypress      24% 206s
[TITLE]  Downloading Cypress      24% 207s
[TITLE]  Downloading Cypress      24% 208s
[TITLE]  Downloading Cypress      25% 198s
[TITLE]  Downloading Cypress      25% 199s
[TITLE]  Downloading Cypress      25% 200s
[TITLE]  Downloading Cypress      25% 201s
[TITLE]  Downloading Cypress      26% 191s
[TITLE]  Downloading Cypress      26% 192s
[TITLE]  Downloading Cypress      26% 194s
[TITLE]  Downloading Cypress      26% 196s
[TITLE]  Downloading Cypress      26% 197s
[TITLE]  Downloading Cypress      27% 187s
[TITLE]  Downloading Cypress      27% 188s
[TITLE]  Downloading Cypress      27% 189s
[TITLE]  Downloading Cypress      27% 190s
[TITLE]  Downloading Cypress      28% 181s
[TITLE]  Downloading Cypress      28% 182s
[TITLE]  Downloading Cypress      28% 183s
[TITLE]  Downloading Cypress      28% 184s
[TITLE]  Downloading Cypress      28% 185s
[TITLE]  Downloading Cypress      29% 176s
[TITLE]  Downloading Cypress      29% 177s
[TITLE]  Downloading Cypress      29% 178s
[TITLE]  Downloading Cypress      29% 179s
[TITLE]  Downloading Cypress      29% 180s
[TITLE]  Downloading Cypress      29% 181s
[TITLE]  Downloading Cypress      29% 182s
[TITLE]  Downloading Cypress      30% 173s
[TITLE]  Downloading Cypress      30% 174s
[TITLE]  Downloading Cypress      30% 175s
[TITLE]  Downloading Cypress      30% 176s
[TITLE]  Downloading Cypress      30% 177s
[TITLE]  Downloading Cypress      30% 178s
[TITLE]  Downloading Cypress      31% 170s
[TITLE]  Downloading Cypress      31% 171s
[TITLE]  Downloading Cypress      31% 172s
[TITLE]  Downloading Cypress      31% 173s
[TITLE]  Downloading Cypress      32% 166s
[TITLE]  Downloading Cypress      32% 167s
[TITLE]  Downloading Cypress      32% 168s
[TITLE]  Downloading Cypress      33% 161s
[TITLE]  Downloading Cypress      33% 162s
[TITLE]  Downloading Cypress      33% 163s
[TITLE]  Downloading Cypress      33% 164s
[TITLE]  Downloading Cypress      34% 157s
[TITLE]  Downloading Cypress      34% 158s
[TITLE]  Downloading Cypress      34% 159s
[TITLE]  Downloading Cypress      35% 153s
[TITLE]  Downloading Cypress      35% 154s
[TITLE]  Downloading Cypress      35% 155s
[TITLE]  Downloading Cypress      35% 156s
[TITLE]  Downloading Cypress      36% 150s
[TITLE]  Downloading Cypress      36% 151s
[TITLE]  Downloading Cypress      36% 152s
[TITLE]  Downloading Cypress      37% 146s
[TITLE]  Downloading Cypress      37% 147s
[TITLE]  Downloading Cypress      37% 148s
[TITLE]  Downloading Cypress      37% 149s
[TITLE]  Downloading Cypress      38% 143s
[TITLE]  Downloading Cypress      38% 144s
[TITLE]  Downloading Cypress      38% 145s
[TITLE]  Downloading Cypress      39% 139s
[TITLE]  Downloading Cypress      39% 140s
[TITLE]  Downloading Cypress      39% 141s
[TITLE]  Downloading Cypress      40% 136s
[TITLE]  Downloading Cypress      40% 137s
[TITLE]  Downloading Cypress      40% 138s
[TITLE]  Downloading Cypress      40% 139s
[TITLE]  Downloading Cypress      41% 134s
[TITLE]  Downloading Cypress      41% 135s
[TITLE]  Downloading Cypress      42% 130s
[TITLE]  Downloading Cypress      42% 131s
[TITLE]  Downloading Cypress      43% 126s
[TITLE]  Downloading Cypress      43% 127s
[TITLE]  Downloading Cypress      43% 128s
[TITLE]  Downloading Cypress      44% 123s
[TITLE]  Downloading Cypress      44% 124s
[TITLE]  Downloading Cypress      44% 125s
[TITLE]  Downloading Cypress      45% 120s
[TITLE]  Downloading Cypress      45% 121s
[TITLE]  Downloading Cypress      45% 122s
[TITLE]  Downloading Cypress      46% 118s
[TITLE]  Downloading Cypress      47% 114s
[TITLE]  Downloading Cypress      47% 115s
[TITLE]  Downloading Cypress      48% 111s
[TITLE]  Downloading Cypress      48% 112s
[TITLE]  Downloading Cypress      49% 107s
[TITLE]  Downloading Cypress      49% 108s
[TITLE]  Downloading Cypress      49% 109s
[TITLE]  Downloading Cypress      50% 105s
[TITLE]  Downloading Cypress      51% 101s
[TITLE]  Downloading Cypress      51% 102s
[TITLE]  Downloading Cypress      52% 98s
[TITLE]  Downloading Cypress      52% 99s
[TITLE]  Downloading Cypress      52% 100s
[TITLE]  Downloading Cypress      53% 96s
[TITLE]  Downloading Cypress      53% 97s
[TITLE]  Downloading Cypress      54% 93s
[TITLE]  Downloading Cypress      54% 94s
[TITLE]  Downloading Cypress      55% 90s
[TITLE]  Downloading Cypress      55% 91s
[TITLE]  Downloading Cypress      56% 88s
[TITLE]  Downloading Cypress      56% 89s
[TITLE]  Downloading Cypress      57% 85s
[TITLE]  Downloading Cypress      57% 86s
[TITLE]  Downloading Cypress      58% 83s
[TITLE]  Downloading Cypress      59% 80s
[TITLE]  Downloading Cypress      59% 81s
[TITLE]  Downloading Cypress      60% 78s
[TITLE]  Downloading Cypress      61% 75s
[TITLE]  Downloading Cypress      62% 72s
[TITLE]  Downloading Cypress      62% 73s
[TITLE]  Downloading Cypress      63% 70s
[TITLE]  Downloading Cypress      63% 71s
[TITLE]  Downloading Cypress      64% 68s
[TITLE]  Downloading Cypress      65% 65s
[TITLE]  Downloading Cypress      65% 66s
[TITLE]  Downloading Cypress      66% 63s
[TITLE]  Downloading Cypress      67% 60s
[TITLE]  Downloading Cypress      67% 61s
[TITLE]  Downloading Cypress      68% 58s
[TITLE]  Downloading Cypress      68% 59s
[TITLE]  Downloading Cypress      69% 56s
[TITLE]  Downloading Cypress      70% 54s
[TITLE]  Downloading Cypress      71% 52s
[TITLE]  Downloading Cypress      72% 50s
[TITLE]  Downloading Cypress      73% 48s
[TITLE]  Downloading Cypress      74% 46s
[TITLE]  Downloading Cypress      75% 44s
[TITLE]  Downloading Cypress      76% 42s
[TITLE]  Downloading Cypress      77% 40s
[TITLE]  Downloading Cypress      78% 38s
[TITLE]  Downloading Cypress      78% 39s
[TITLE]  Downloading Cypress      79% 36s
[TITLE]  Downloading Cypress      79% 37s
[TITLE]  Downloading Cypress      80% 34s
[TITLE]  Downloading Cypress      80% 35s
[TITLE]  Downloading Cypress      81% 32s
[TITLE]  Downloading Cypress      81% 33s
[TITLE]  Downloading Cypress      82% 31s
[TITLE]  Downloading Cypress      83% 29s
[TITLE]  Downloading Cypress      84% 27s
[TITLE]  Downloading Cypress      85% 25s
[TITLE]  Downloading Cypress      86% 23s
[TITLE]  Downloading Cypress      86% 24s
[TITLE]  Downloading Cypress      87% 22s
[TITLE]  Downloading Cypress      88% 20s
[TITLE]  Downloading Cypress      89% 18s
[TITLE]  Downloading Cypress      90% 17s
[TITLE]  Downloading Cypress      91% 15s
[TITLE]  Downloading Cypress      92% 13s
[TITLE]  Downloading Cypress      93% 11s
[TITLE]  Downloading Cypress      93% 12s
[TITLE]  Downloading Cypress      94% 10s
[TITLE]  Downloading Cypress      95% 8s
[TITLE]  Downloading Cypress      96% 6s
[TITLE]  Downloading Cypress      97% 5s
[TITLE]  Downloading Cypress      98% 3s
[TITLE]  Downloading Cypress      99% 2s
[TITLE]  Downloading Cypress      100% 0s
[TITLE]  Downloaded Cypress     
[SUCCESS]  Downloaded Cypress     
[STARTED] Task without title.
[FAILED] The Cypress App could not be unzipped.
[FAILED] 
[FAILED] Search for an existing issue or open a GitHub issue at
[FAILED] 
[FAILED] https://github.com/cypress-io/cypress/issues
[FAILED] 
[FAILED] ----------
[FAILED] 
[FAILED] Error: EACCES: permission denied, mkdir '/Users/mac/Library/Caches/Cypress/11.2.0'
[FAILED] 
[FAILED] ----------
[FAILED] 
[FAILED] Platform: darwin-x64 (20.2.0)
[FAILED] Cypress Version: 11.2.0
The Cypress App could not be unzipped.

Search for an existing issue or open a GitHub issue at

https://github.com/cypress-io/cypress/issues

----------

Error: EACCES: permission denied, mkdir '/Users/mac/Library/Caches/Cypress/11.2.0'

----------

Platform: darwin-x64 (20.2.0)
Cypress Version: 11.2.0
error Command failed with exit code 1.
error Command failed with exit code 1.

**Comment by portyu9** [comment]
Hi @scriptlord 

The download itself is succeeding. The important error is:

```text
EACCES: permission denied, mkdir '/Users/mac/Library/Caches/Cypress/11.2.0'
```

So this is a **filesystem permission problem in the Cypress binary cache**, not a corrupt Cypress ZIP and not an Nx/Angular generation problem.

Check the ownership first:

```bash
ls -ld "$HOME/Library/Caches/Cypress"
```

If that directory was accidentally created by `root` or another user, fix its ownership:

```bash
sudo chown -R "$(id -un)":"$(id -gn)" "$HOME/Library/Caches/Cypress"
```

Then retry the install.

Alternatively, tell Cypress to use another writable directory:

```bash
export CYPRESS_CACHE_FOLDER="$HOME/.cache/Cypress"
mkdir -p "$CYPRESS_CACHE_FOLDER"

yarn install
```

or, for a project-local cache:

```bash
export CYPRESS_CACHE_FOLDER="$PWD/.cache/Cypress"
mkdir -p "$CYPRESS_CACHE_FOLDER"

yarn install
```

Keep the same `CYPRESS_CACHE_FOLDER` setting when Cypress is later launched.

I would **not** solve this by running `npm`, `yarn`, or Nx itself with `sudo`; that commonly creates more root-owned files and causes additional permission problems later.

Please mark this answer as accepted if it helped, thank you.
